Algorithm/QCAlgorithm.cs

```diff
@@ -539,6 +548,35 @@ public void SetParameters(Dictionary<string, string> parameters)
         }
 
         /// <summary>
+        /// Set the available data feeds in the <see cref="SecurityManager"/>
+        /// </summary>
+        /// <param name="availableDataFeeds">The different <see cref="TickType"/> each <see cref="Security"/> supports</param>
+        public void SetAvailableDataFeeds(Dictionary<SecurityType, List<TickType>> availableDataFeeds)
+        {
+            foreach (var dataFeed in availableDataFeeds)
+            {
+                AvailableDataFeeds[dataFeed.Key] = dataFeed.Value;
+            }
+        }
+
+        /// <summary>
+        /// Hard code the set of default available data feeds
+        /// </summary>
+        public Dictionary<SecurityType, List<TickType>> DefaultDataFeeds()
```

Comment:
Hey Andrew, 

Thanks for the PR. 

Wondering what is the idea behind these default data feeds? Currently we have tradebars, quotebars, oi supported in futures, options. Should we reflect it in the default list or use the file with overrides (via "security-data-feeds")?

I think it makes sense to fully test this in local live/backtest configurations for futures/options.